### DISCUSSION TITLE
How to clean/reset controller mapping

### DISCUSSION THREAD

**Discussion by jsteerv** [AntiMicroX/antimicrox#213] [Q&A] [answered]
**Question**

Hi, I am using an Xbox controller connected through Bluetooth. It was working until I assigned Ctrl key to the right throttle then the controller gets frozen, I tried restarting everything but now I can't get the controller connected, the Xbox led is intermittent, and the connection process restarts indefinitely.

I am using:
- ubuntu 20.04
- antimicrox 3.1.5

Is there any way to clean/reset the key mapping using the terminal before connecting the controller?

Thanks in advance!

**Comment by vguttmann** [comment, marked as answer]
So, in case that's still an issue for you, here's what you need to do: Find your current controller mappings, and remove them. Unfortunately, you will have to redo your mapping, but that's what it takes I guess.
So, now to finding your mappings: AntiMicroX stores the mappings in .amgp files. So, to find those, you need to run `find / -name *.amgp`. 
Then, you can basically delete the mapping that causes the freeze, because there is no easy way of recovering that. If you're not sure, move them to a thumb drive one by one, and delete the one that seemed to cause the issue.

**Comment by jsteerv** [comment]
Thank you @vguttmann it worked!!

Very helpful, wonder if it could be included in the FAQ or README.
